src/EventListener/SerializeListener.php

```diff
@@ -47,7 +47,7 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
         $controllerResult = $event->getControllerResult();
         $request = $event->getRequest();
 
-        if ($controllerResult instanceof Response) {
+        if ($controllerResult instanceof Response || !$request->attributes->getBoolean('_api_respond', true)) {
```

Comment:
might break things no?

Reply:
These changes are not even from me, I created the pull request from the wrong point. But this is now fixed.